canonical-web-and-design/ubuntu.com
# Telco contact form

## Done

- Created Telco contact form
- Created Thank-you page and base_telco

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1kAY2ScECR-auRINeK6Ve41_M8liHZqRMOWwGXWqEYuw/edit)
- When new telco page is live, check the form opens and submits properly.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4194
